### PR TITLE
Improvement: Test Coverage Improvement for Handling Valid Properties

### PR DESCRIPTION
**Pull request by Chinex-Boroja**
### Overview

This is a pull request raised for issue [#447](https://github.com/odk-x/tool-suite-X/issues/447)

### Reason for improvement

It can be observed that current tests in the class only set and retrieve simple properties correctly. Also, it checks for secure properties. So an improvement has been made to check if the application handles data correctly within its boundaries. This test case checks if the application handles font size correctly and manages edge cases related to font sizes.

### Changes made to the `PropertiesTest.java` class

- Added a test to check if the application handles font sizes correctly and manages edge cases related to the font sizes
- Added property to set the `username`, retrieved the `username` property, and checked if the property was set correctly
- Refactored `InstrumentationRegistry.getContext()` to `InstrumentationRegistry.getInstrumentation().getTargetContext()`

### Screenshot of Test Cases Passed
testFontSizeEdgeCases

### CODE REVIEW

**Comment by wbrunette** [comment]
@Chinex-Boroja please resolve your conflicts after changes. It demonstrates your Git knowledge :)

**Comment by Chinex-Boroja** [comment]
> @Chinex-Boroja please resolve your conflicts after changes. It demonstrates your Git knowledge :)

@wbrunette I am done resolving the conflicts. Kindly review. Thank you

**Comment by wbrunette** [comment]
This PR looks good now. We won't merge it until afterwards since some contributions are similar for other PRs (not your test, but the deprecated functions). As once it's merged the removal of deprecated functions might disappear.

**Review by wbrunette — APPROVED** [review]

**Pull request merged**
